recipe/build.sh

```diff
@@ -1,39 +1,35 @@
 #!/bin/bash -euo
 
 chmod +x bin/*
-chmod +x jre/bin/*
 mv bin/* $PREFIX/bin/
 ls -la $PREFIX/bin
+
 mv include/* $PREFIX/include
-if [ -e jre/lib/jspawnhelper ]; then
-    chmod +x jre/lib/jspawnhelper
+if [ -e ./lib/jspawnhelper ]; then
+    chmod +x ./lib/jspawnhelper
 fi
 
 if [[ `uname` == "Linux" ]]
 then
-    mv lib/amd64/jli/*.so lib
-    mv lib/amd64/*.so lib
-    rm -r lib/amd64
-    # libnio.so does not find this within jre/lib/amd64 subdirectory
-    cp jre/lib/amd64/libnet.so lib
+    mv lib/jli/*.so lib/
 
-    # include dejavu fonts to allow java to work even on minimal cloud images where these fonts are missing
-    # (thanks to @chapmanb)
-    mkdir -p jre/lib/fonts
-    cd jre/lib/fonts
+    # Include dejavu fonts to allow java to work even on minimal cloud
+    # images where these fonts are missing (thanks to @chapmanb)
+    mkdir -p lib/fonts
+    cd lib/fonts
     curl -L -O -C - http://sourceforge.net/projects/dejavu/files/dejavu/2.36/dejavu-fonts-ttf-2.36.tar.bz2
```

Comment:
Please use the multiple sources capability from conda-build-3 to download and unpack the dejavu files. 

See [here]( https://github.com/conda-forge/gosu-feedstock/blob/c7fc95d0fd963b13dd8601d5d9adf08d464f0de4/recipe/meta.yaml#L11-L17) for an example.